lib/vtls/schannel.c

```diff
@@ -171,7 +180,7 @@ static void InitSecBufferDesc(SecBufferDesc *desc, SecBuffer *BufArr,
 }
 
 static CURLcode
-set_ssl_version_min_max(SCHANNEL_CRED *schannel_cred, struct Curl_easy *data,
+set_ssl_version_min_max(DWORD *grbitEnabledProtocols, struct Curl_easy *data,
```

Comment:
`grbitEnabledProtocols` is a peculiar-looking name that stands out next to our regular curl names. I understand you use this because that's the name of the struct field this ultimately gets assigned to. I still believe just `enabledprotocols` for example would read much nicer.

Reply:
You're right. I'll make this change.